### PR TITLE
fix: Add enn-dee

### LINKED ISSUE #1176: enn dee
## Profile Screenshot
13 06 2022_19 03 03_REC

### PR DESCRIPTION
**Pull request by enn-dee**
resolves #1176

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## Notes

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

### CODE REVIEW

**Review by MrKrishnaAgarwal — APPROVED** [review]
Looks Great👍🏻

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey!

The issue number #1168  you provided is opened but someone else, and I can't find an issue opened by you. Please open an issue and also modify the image link as well. 

If you have any doubt feel free to reach out



__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by MrKrishnaAgarwal** [comment]
> Hey!
> 
> The issue number #1168 you provided is opened but someone else, and I can't find an issue opened by you. Please open an issue and also modify the image link as well.
> 
> If you have any doubt feel free to reach out
> 
> __ [sema-logo](https://semasoftware.com/gh) [ sema-logo ](https://semasoftware.com/gh) [ ](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

Oh Yes, I didn't saw that one.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge, thank you for your contribution!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by MrKrishnaAgarwal — APPROVED** [review]
Your Profile Looks Great 👍
All the best for the future
Keep Contributing
Also thanks for giving me the chance for reviewing

File: profiles/enn-dee.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/89350768/173366815-a919027a-2c53-4b9d-a029-ef0dbb2ede27.png",
```
Comment (Panquesito7) [inline profiles/enn-dee.json:2]: It looks like the image link is different from the issue. Do you want to change it?
```suggestion
  "image": "https://user-images.githubusercontent.com/89350768/173365612-f5ddec29-c3f8-4bc4-a616-d9d0fa2ea6fd.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

**Review by kabir0x23 — APPROVED** [review]
Looks good 👍
Nice profile 🚀

**Pull request merged**